test/unit/reducer/test-reduce-constructors.cpp

```diff
@@ -0,0 +1,221 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-19, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+///
+/// Source file containing tests for RAJA reducer constructors and initialization.
+///
+
+#include "gtest/gtest.h"
+
+#include "RAJA/RAJA.hpp"
+#include "RAJA_gtest.hpp"
+#include "RAJA/internal/MemUtils_CPU.hpp"
+
+#if defined(RAJA_ENABLE_CUDA)
+#include "RAJA_unit_forone.hpp"
+#endif
+
+#include <tuple>
+
+template <typename T>
+class ReducerBasicConstructorTest : public ::testing::Test
+{
+};
+
+template <typename T>
+class ReducerInitConstructorTest : public ::testing::Test
+{
+};
+
+#if defined(RAJA_ENABLE_CUDA)
+template <typename T>
+class ReducerCUDAConstructorTest : public ::testing::Test
+{
+};
+#endif
+
+TYPED_TEST_CASE_P(ReducerBasicConstructorTest);
```

Comment:
Please replace 'Test' with 'UnitTest' in all test labels.

Reply:
Thanks, this should be updated now.